lib/miq_expression.rb

```diff
@@ -1172,11 +1172,11 @@ def self._custom_details_for(model, options)
     custom_attributes_details = []
 
     klass.custom_keys.each do |custom_key|
-      custom_detail_column = [model, CustomAttributeMixin::CUSTOM_ATTRIBUTES_PREFIX + custom_key].join("-")
-      custom_detail_name = custom_key
+      custom_detail_column = [model, CustomAttributeMixin::CUSTOM_ATTRIBUTES_PREFIX + custom_key[1]].join("-")
```

Comment:
Instead of array indexing, it might be more readable to do 

```ruby
klass.custom_keys.each do |custom_section, custom_key|
```

and then use the named variables.